astropy/cosmology/io/yaml.py

```diff
@@ -53,13 +53,15 @@ def representer(dumper, obj):
             :mod:`yaml` representation of |Cosmology| object.
         """
         # convert to mapping
-        map = obj.to_format("mapping")
```

Comment:
I would discourage using one-character variable names except in limited cases like a list comprehension or standard index variables like `i, j, k`.  (And personally I often use `ii` just because I find it easier to read, but that is not really standard).

Reply:
SGTM. I'll revert.